project/ScalazPlugin.scala

```diff
@@ -19,7 +19,7 @@ object ScalazPlugin extends AutoPlugin {
   override def trigger = allRequirements
 
   override lazy val projectSettings = Seq(
-    scalazVersion := sys.env.get("SCALAZ_VERSION").getOrElse("7.2.15"),
+    scalazVersion := sys.env.get("SCALAZ_VERSION").getOrElse("7.2.17"),
```

Comment:
Non-extant scalaz version... or if it does you are ahead of their release page -  `7.2.16`

Reply:
Typo.